python/UniverseWrapper.cpp

```diff
@@ -239,6 +241,23 @@ namespace {
     { return rules.RuleExists(name); }
     bool RuleExistsWithType(const GameRules& rules, const std::string& name, GameRules::Type type)
     { return rules.RuleExists(name, type); }
+
+    boost::python::object GetOptionsDBOptionStr(std::string const &option)
```

Comment:
`std::string const &option` -> `const std::string& option`

Reply:
It is copypaste from AIWrapper